conf/nginx.conf.server.template

```diff
@@ -8,7 +8,7 @@ proxy_cache_methods GET HEAD OPTIONS;
 proxy_cache_lock on;
 
 ## Ignore headers that make requests uncacheable
-proxy_ignore_headers Set-Cookie Vary X-Accel-Buffering;
+proxy_ignore_headers Set-Cookie Vary;
```

Comment:
For some context: We found that SSE is not working on the device (specifically over HTTPs) unless we add `X-Accel-Buffering: no` to the response headers. 

However here we are telling nginx to ignore this header so ever after adding it SSE still doesn't work. 

It's not clear to me if/how `X-Accel-Buffering` is uncacheable (according to the comment above). I get why Set-Cookie and Vary are not, but I don't know how `X-Accel-Buffering` affects the cache.

Reply:
Hmm, is it possible that some servers set X-Accel-Buffering by accident?  Seems like a common misconfiguration - someone sets this header to control their proxy and accidentally forwards it to us, and it seems reasonable to me that we'd ignore it.

Reply:
Thanks, that makes sense. I'm in favour of removing `X-Accel-Buffering` from `proxy_ignore_headers` as I think a better default is to respect what the response is telling us to do. We definitely need to do some testing though to make sure this doesn't break any core or important customer apps.